examples/sharding.js

```diff
@@ -4,6 +4,7 @@ const Eris = require("eris");
 const bot = new Eris("Bot TOKEN", {
     firstShardID: 0,
     lastShardID: 15,
+    shardIDs: [5, 9, 1, 4], // Overrides firstShardID and lastShardID above
```

Comment:
it might be worth noting shards will connect in the order specified
![image](https://user-images.githubusercontent.com/17226394/174496232-e0a2763f-89d5-49c2-bba7-3afd7a90c7ab.png)

Reply:
It's a bit more nuanced then that. They are initialized in the order specified however concurrency will start them as fast as possible per bucket so the order is random in that case.